src/virtual_dom/vcomp.rs

```diff
@@ -266,3 +270,14 @@ impl<COMP: Component> fmt::Debug for VChild<COMP> {
         f.write_str("VChild<_>")
     }
 }
+
+impl ToHtml for VComp {
+    fn to_html(&self) -> String {
+        // htmlescape
+
+        match *((*self.state).borrow()) {
+            MountState::Mounted(mounted) => mounted.to_string(),
```

Comment:
I'm currently getting the error here
```
84  | struct Mounted {
    | -------------- method `to_string` not found for this
...
280 |             MountState::Mounted(mounted) => mounted.to_string(),
    |                                                     ^^^^^^^^^ method not found in `virtual_dom::vcomp::Mounted`
```

I understand that this is part of the missing plumbing peace you've mentioned, correct? I think we could introduce a state here which is rendered but unmounted. I'm not sure if we would consider them mutually exclusive. Probably if we have a state `RenderedInternally` vs `Mounted`, as `Mounted` includes `RenderedByBrowser` IIUC.

Reply:
That could work, and interesting question.